automated_testing/testcases/send_file/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM habitat_integration_base
+
+# Always accept the license when we run this image.
+ENV HAB_LICENSE=accept-no-persist
+
+# We're going to be using docker inside the container.
+#
+# TODO: Consider ensuring that the base container itself has Docker if
+# this becomes more of a pattern.
+RUN hab pkg install core/docker -b -f
```

Comment:
I think it would be nice for this to "just be there". In testing this PR with rolling builds I have used docker in a few tests.

Reply:
Will do.